client/lib/analytics/ad-tracking.js

```diff
@@ -197,13 +197,28 @@ function loadTrackingScripts( callback ) {
 }
 
 /**
- * Whether we're allowed to track the current page for retargeting.
+ * Returns whether ad tracking is allowed.
  *
- * We disable retargetinig on certain pages/URLs in order to protect our users' privacy.
+ * This function returns false if:
+ *
+ * 1. 'ad-tracking' is disabled
+ * 2. `Do Not Track` is enabled
+ * 3. `document.location.href` may contain personally identifiable information
  *
  * @returns {Boolean}
  */
-function isRetargetAllowed() {
+function isAdTrackingAllowed() {
+	// 1. 'ad-tracking' is disabled
+	if ( ! config.isEnabled( 'ad-tracking' ) ) {
+		return false;
+	}
+
+	// 2. User has Do Not Track enabled
+	if ( '1' === navigator.doNotTrack ) {
```

Comment:
this is used in a couple of places - maybe define a `doNotTrack` function in `client/lib/analytics/doNotTrack.js`